src/components/jellyfinActions.ts

```diff
@@ -242,7 +242,7 @@ function setRandomUserBackdrop(): Promise<void> {
             // not everyone will want to see adult backdrops rotating on their TV.
             MaxOfficialRating: 'PG-13'
         }
-    }).then(function (result: BaseItemDtoQueryResult) {
+    }).then((result: BaseItemDtoQueryResult) => {
```

Comment:
Can you fix the new lint errors as well? Ideally PRs that fail checks should not be merged, hence I at least expect the new linting checks to also be corrected for by this PR. If you have time, the other linting checks as well.

In this instance this function should be rewritten to await syntax.